sections/sprint-4/updated-uppaal.tex

```diff
@@ -119,26 +126,26 @@ \subsection{UDP host templates}
 \end{figure}
 
 % Trace example of the system
-\subsection{Happy simulation trace}
-A happy simulation trace can be seen on \autoref{fig:sprint4-happy-trace}.
+\subsection{Perfect scenario simulation trace}
+A simulation trace where everything goes perfectly and no timeouts happen can be seen on \autoref{fig:sprint4-happy-trace}.
 As seen on the trace \uppTemp{Client(0)} first connects through the threeway handshake without any timeouts.
-\uppTemp{Client(1)} Then also connects through threeway handshake without any timeouts.
-Finally the after both of the clients are in \uppLoc{Lobby} the \uppTemp{Host} transitions to \uppLoc{InGame}.
+\uppTemp{Client(1)} then also connects through three-way handshake without any timeouts.
+Finally, after both of the clients are in the \uppLoc{Lobby}, the \uppTemp{Host} transitions to \uppLoc{InGame}.
 \begin{figure}[h]
     \centering
     \includegraphics[width=1\linewidth]{sprint4/connected-trace.png}
-    \caption{Example of a trace where both Clients connect without any problems}
+    \caption{Example of a trace where both Clients connect without any problems.}
     \label{fig:sprint4-happy-trace}
 \end{figure}
 
 \subsection{Trace with a timeout}
 A trace where it times out once while connecting can be seen on \autoref{fig:sprint4-timeout-trace}.
-\uppTemp{Client(1)} connects without any issues, but \uppTemp{Client(0)} times out once while connecing.
-This can be seen on the trace of \uppTemp{TCP_client(0)}. When it sends the \uppOut{tcp_sync[0]} it timesout because it doesn't get are response from the \uppTemp{TCP_host(1)}.
+\uppTemp{Client(1)} connects without any issues, but \uppTemp{Client(0)} times out once while connecting.
+This can be seen on the trace of \uppTemp{TCP_client(0)}. When it sends the \uppOut{tcp_sync[0]} it times out because it does not get a response from the \uppTemp{TCP_host(1)}.
```

Comment:
```suggestion
This can be seen on the trace of \uppProc{TCP_client(0)}. When it sends the \uppOut{tcp_sync[0]} it times out because it does not get a response from the \uppProc{TCP_host(1)}.
```